tests/puppeteer/campaigns/functional/BO/04_customers/01_customers/08_viewCustomer.js

```diff
@@ -0,0 +1,529 @@
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAddNewCustomerPage', baseContext);
+      await this.pageObjects.customersPage.goToAddNewCustomerPage();
+      const pageTitle = await this.pageObjects.addCustomerPage.getPageTitle();
+      await expect(pageTitle).to.contains(this.pageObjects.addCustomerPage.pageTitleCreate);
+    });
+
+    it('should create customer and check result', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'createCustomer', baseContext);
+      const textResult = await this.pageObjects.addCustomerPage.createEditCustomer(createCustomerData);
+      await expect(textResult).to.equal(this.pageObjects.customersPage.successfulCreationMessage);
+      const numberOfCustomersAfterCreation = await this.pageObjects.customersPage.getNumberOfElementInGrid();
+      await expect(numberOfCustomersAfterCreation).to.be.equal(numberOfCustomers + 1);
+    });
+  });
+  // 2 : View customer
+  describe('View customer created', async () => {
+    it('should filter list by email', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterToViewCreatedCustomer', baseContext);
+      await this.pageObjects.customersPage.resetFilter();
+      await this.pageObjects.customersPage.filterCustomers(
+        'input',
+        'email',
+        createCustomerData.email,
+      );
+      const textEmail = await this.pageObjects.customersPage.getTextColumnFromTableCustomers(1, 'email');
+      await expect(textEmail).to.contains(createCustomerData.email);
+    });
+
+    it('should click on view customer', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToViewCustomerPageAfterCreateCustomer', baseContext);
+      await this.pageObjects.customersPage.goToViewCustomerPage(1);
+      const pageTitle = await this.pageObjects.viewCustomerPage.getPageTitle();
+      await expect(pageTitle).to.contains(this.pageObjects.viewCustomerPage.pageTitle);
+    });
+
+    it('should check personal information title', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkPersonalInformationTitle1', baseContext);
+      const cardHeaderText = await this.pageObjects.viewCustomerPage.getPersonalInformationTitle();
+      await expect(cardHeaderText).to.contains(createCustomerData.firstName);
+      await expect(cardHeaderText).to.contains(createCustomerData.lastName);
+      await expect(cardHeaderText).to.contains(createCustomerData.email);
+    });
+
+    it('should check customer personal information', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'checkCreatedCustomerInfo1', baseContext);
+      const cardHeaderText = await this.pageObjects.viewCustomerPage.getTextFromElement('Personal information');
+      await expect(cardHeaderText).to.contains(createCustomerData.socialTitle);
+      await expect(cardHeaderText).to.contains(`birth date: ${customerBirthDate}`);
```

Comment:
Not began by a uppercase ?

Reply:
No it not began by an uppercase in the form

Reply:
:ok_hand: